yarn-project/simulator/src/client/execution_result.test.ts

```diff
@@ -1,222 +0,0 @@
-import {
```

Comment:
Removed this file as it only tested the collect logs functionality, which is now removed.